.circleci/config.yml

```diff
@@ -1,151 +1,28 @@
 version: 2.1
-
-orbs:
-  aws-s3: circleci/aws-s3@1.0.11
-
-only_branches: &only_branches
-  filters:
-    branches:
-      ignore: master
-
-only_master: &only_master
-  filters:
-    branches:
-      only: master
-
-only_v_tags: &only_v_tags
-  filters:
-    tags:
-      only: /^v.*/
-    branches:
-      ignore: /.*/
-
-mm_binary: &mm_binary
-  mm-binary: https://releases.mattermost.com/mattermost-platform/master/mattermost-enterprise-linux-amd64.tar.gz
+executors:
+  default:
+    docker:
+      - image: circleci/golang:1.13
 
 jobs:
-  build-ltops-and-test:
-    docker:
-      - image: circleci/golang:1.12
+  lint:
+    executor:
+      name: default
     steps:
       - checkout
-      - restore_cache:
-          key: dependencies-{{ checksum "go.sum" }}
-      - run:
-          name: Install dependencies
-          command: |
-            export PATH=$PATH:$GOPATH/bin
-            mkdir -p $GOPATH/bin
-            make install
-      - save_cache:
-          key: dependencies-{{ checksum "go.sum" }}
-          paths:
-            - $GOPATH/pkg/mod
-      - run:
-          name: Build package
-          command: make package
-      - run:
-          name: Test
-          command: |
-            set +x
-            ltops --help
-      - persist_to_workspace:
-          root: .
-          paths:
-            - dist
+      - run: curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.21.0
```

Comment:
@lieut-data did a great job in separating the install golangci-lint step into a separate job: https://github.com/mattermost/mattermost-marketplace/blob/b7089abfd665de995d8cee45ff5f7586b42f850c/.circleci/config.yml#L30. It also uses GOBIN rather than GOPATH/bin to install the binary which is preferred, since we are moving away from all GOPATH related references.

Let's do the same here.